tests/cosmogony_lux_test.rs

```diff
@@ -1,7 +1,11 @@
 extern crate cosmogony;
+extern crate geo;
```

Comment:
are you sure this is needed ? :thinking:

Reply:
It is not indeed, good catch !